### PR TITLE
weekly update

### PR DESCRIPTION
**Pull request by jxue16**

### CODE REVIEW

File: assets/ai2.yaml [hunk 1]
```diff
@@ -115,3 +115,97 @@
...
+  created_date: 2023-11-20
+  url: https://arxiv.org/pdf/2311.10702.pdf
+  datasheet: https://huggingface.co/datasets/allenai/tulu-v2-sft-mixture
+  modality: text
+  size:
+    value: unknown
+    explanation: Magnitude of size is around 100M tokens, given the length distribution of dataset provided in model card.
+  sample: []
+  analysis: Models trained with dataset evaluated on downstream performance.
+  dependencies: [FLAN, Open Assistant 1, ShareGPT, GPT4 Alpaca, Code Alpaca, LIMA, WizardLM Evol Instruct, Open Orca]
```
Comment (rishibommasani) [inline assets/ai2.yaml:132]: Can you update these names so they match the existing nodes we have, so the graph gets connected correctly? E.g. I am guessing GPT4 Alpaca should be Alpaca dataset?

File: assets/inflection.yaml [hunk 2]
```diff
@@ -45,3 +45,28 @@
...
+  organization: Inflection AI
+  description: Inflection-2 is the best model in the world for its compute class and the second most capable LLM in the world, according to benchmark evaluation, as of its release.
+  created_date: 2023-11-22
+  url: https://inflection.ai/inflection-2
+  model_card: none
+  modality: text; text
+  analysis: Evaluated against state of the art models on benchmarks, and found to be most performant model outside of GPT-4.
+  size:
+    value: unknown
+    explanation: Likely larger than Inflection-1, which was stated to be around the size of ChatGPT (175B parameters).
```
Comment (rishibommasani) [inline assets/inflection.yaml:60]: Where is this from? I don't think this is true if I were to guess

File: assets/inflection.yaml [hunk 3]
```diff
@@ -45,3 +45,28 @@
...
+  dependencies: []
+  training_emissions: unknown
+  training_time: unknown
+  training_hardware: 5000 NVIDIA H100 GPUs
+  quality_control: ''
+  access: limited
```
Comment (rishibommasani) [inline assets/inflection.yaml:66]: is there access to inflection 2?

Comment (rishibommasani) [reply assets/inflection.yaml:66]: i would have thought it is closed

File: assets/meta.yaml [hunk 4]
```diff
@@ -694,3 +694,26 @@
   prohibited_uses: ''
   monitoring: ''
   feedback: ''
+
+- type: model
+  name: Code LLaMA
```
Comment (rishibommasani) [inline assets/meta.yaml:699]: were we missing this/why is this added in this PR?

**Review by rishibommasani — CHANGES_REQUESTED** [review]

File: assets/xai.yaml [hunk 5]
```diff
@@ -0,0 +1,24 @@
+---
+- type: model
+  name: Grok-1
+  organization: xAI
+  description: Grok is a text chatbot modeled after the Hitchhiker’s Guide to the Galaxy, so intended to answer almost anything and even suggest what questions to ask.
+  created_date: 2023-11-04
+  url: https://github.com/01-ai/Yi
```
Comment (rishibommasani) [inline assets/xai.yaml:7]: wrong urk

**Review by rishibommasani — CHANGES_REQUESTED** [review]
Left some comments

File: assets/inflection.yaml (hunk 2 above)
Comment (jxue16) [reply assets/inflection.yaml:60]: I might be misunderstanding, but in both Inflection 2's release doc it's stated to be in the same compute class as PaLM 2 and Inflection 1's release doc is said to be in the same compute class as GPT-3.5. Am I conflating compute class and model size?

File: assets/meta.yaml (hunk 4 above)
Comment (jxue16) [reply assets/meta.yaml:699]: I couldn't find this previously in EG, and I believe some assets list it as a dependency, so I added it in with the PR. I can remove this if it was already added by a different name?

File: assets/inflection.yaml (hunk 3 above)
Comment (jxue16) [reply assets/inflection.yaml:66]: You're right, it should be closed.

**Review by rishibommasani — CHANGES_REQUESTED** [review]
